central/src/components/AddQuestionSidebar.jsx

```diff
@@ -3,17 +3,56 @@ import { Route, Switch, useHistory } from 'react-router-dom';
 import { makeStyles } from '@material-ui/core/styles';
 import { Grid, Button } from "@material-ui/core";
 import ArrowBack from '@material-ui/icons/ArrowBack';
-import GameDashboard from './GameDashboard';
-import AddQuestion from "./AddQuestion";
-import { getGameById } from "../lib/games";
+import QuestionDashboard from './QuestionDashboard';
+
+export default function AddQuestionForm({ loading, questions, games, cloneQuestion, submit, gameId }) {
+    const classes = useStyles();
+    const history = useHistory();
+
+    return (
+        <Grid container className={classes.root}>
+          <Button type="button" className={classes.back} onClick={() => history.push(`/gamemaker/${gameId}`)}>
+            <ArrowBack style={{marginRight: 8}} />Back to Game Maker
+          </Button>
+
+          <Grid item xs={12} className={classes.sidebar}>
+            <QuestionDashboard questions={questions} />
+          </Grid>
+
+          {/* <Grid item xs={7} className={classes.content}>
+            <Switch>
+              <Route path="/gamemaker/:gameId/addquestion/gameSelected/:selectedId" render={
+                ({ match }) => {
+                  const {gameId, selectedId} = match.params;
+                  return <AddQuestion game={getGameById(games, selectedId)}  cloneQuestion={cloneQuestion} submit={submit} selectedId={selectedId} gameId={gameId} />;
+                }
+              } />
+
+              <Route path="/gamemaker/:gameId/addquestion" render={
+                ({ match }) => {
+                  return (
+                    <Grid style={{height: 'calc(100vh - 64px)'}}>
+                      <p style={{color:"#797979", fontWeight:"bold"}}>No Game Selected</p>
+
+                      <h2 style={{width: "60%",color:"#797979", marginLeft:"auto", marginRight:"auto", marginTop:"30%"}}>In order to view questions, you must select a game from the section on the left</h2>
+                    </Grid>
+                  );
+                }
+              }/>
+            </Switch>           
+          </Grid> */}
```

Comment:
In case not used in another PR, we should remove it

Reply:
removed